app/models/user.rb

```diff
@@ -92,6 +92,23 @@ def admin?
     !!admin
   end
 
+  # TODO: We are getting first_name and last_name from users when the join the union
```

Comment:
Do we have a ticket created for this?
_Make sure to keep this tech debt visible..._

Reply:
https://app.asana.com/0/1195530778765030/1199544593172957